Autopilot/TelemetryManager/Src/telemetryStateClasses.cpp

```diff
@@ -1,12 +1,23 @@
 /*
 * Author: Nixon Chan
+* Implementing Telem Comms: Gordon Fountain
 */
 
 #include "telemetryStateClasses.hpp"
 
+
+Telemetry_PIGO_t _PMData;
+POGI GSData;
```

Comment:
Let's rename some of these for consistency: `POGI --> Telemetry_POGI_t`, `GSData --> _GSData`

Also, these should be declared as `static` so they only have file scope and not global scope ;)

Reply:
Okay, will also change POGI in the telem path interface